sst/packages/functions/src/cron-jobs/govsg-webhook-monitor/cron.ts

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
+import { sql } from 'drizzle-orm'
+import { Config } from 'sst/node/config'
+
+import PostmanDbClient from '@/core/database/client'
+import { govsgMessages } from '@/core/models'
+
+export async function handler() {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const cronitor = require('cronitor')(Config.CRONITOR_URL_SUFFIX) // common to all jobs on our shared Cronitor account
+    const invokeFunction =
+      process.env.IS_LOCAL === 'true'
+        ? async function () {
+            console.log('Running cron locally')
+            await checkMessagesNotGettingWebhooks()
+          }
+        : cronitor.wrap(
+            Config.CRONITOR_CODE_GOVSG_WEBHOOK_MONITOR,
+            async function () {
+              await checkMessagesNotGettingWebhooks()
+            },
+          )
+    await invokeFunction()
+  } catch (error) {
+    console.log(error)
+    throw error
+  }
+}
+
+async function checkMessagesNotGettingWebhooks() {
+  const db = new PostmanDbClient(Config.POSTMAN_DB_URI).getClient()
+  const noWebhookMessages = await db
+    .select({})
+    .from(govsgMessages)
+    .where(
+      // the reason we want to query for the previous 10-minute block instead of
+      // the immediate one is to prevent messages from campaigns currently being sent
+      // out from being flagged (This is assuming a campaign won't take more than
```

Comment:
Is this assumption an estimation derived from Grafana?

Reply:
no so the average webhooks for sent actually comes back much faster (about a few seconds), but I'm just taking 10 minutes just to be safe, and I think our SLA to detect this kind is 30 mins so shud be ok